seed/seed16.go

```diff
@@ -112,6 +131,13 @@ func (s *seed16) Model() (*asserts.Model, error) {
 	return s.model, nil
 }
 
+func (s *seed16) Brand() (*asserts.Account, error) {
+	if s.brand == nil {
```

Comment:
I would just do a s.db.Find here

Reply:
Ah, right, that makes sense. I'll push an update.

Reply:
This is done now.